core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -440,6 +480,7 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
                     this.store.savePage({
                         pageContainers,
                         pageId: payload.pageId,
+                        params: this.queryParams as DotPageApiParams,
```

Comment:
Should we do a getter to assert in one place?

Reply:
Yes! We can do that